docs/visualization/lupton_rgb.rst

```diff
@@ -0,0 +1,26 @@
+**********************************
+Creating color RGB images
+**********************************
+
+`Lupton et al. (2004)`_ describe an "optimal" algorithm for producing red-green-
+blue composite images from three separate high-dynamic range arrays. This method
+is implemented in `astropy.visualization.lupton_rgb` as a set of classes
+providing different scalings and a convenience wraper function. To generate a
+color PNG file with the default (arcsinh) scaling:
+
+>>> from astropy.visualization import lupton_rgb
```

Comment:
CircleCI gives the following error for `import matplotlib.image`:

```UNEXPECTED EXCEPTION: ImportError('No module named matplotlib.image',)```

Is the version of matplotlib that circleCI uses out of date?